wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/homescreen/pager/CertificatePagerFragment.kt

```diff
@@ -95,6 +93,14 @@ class CertificatePagerFragment : Fragment() {
 		_binding = null
 	}
 
+	private fun updateTitle() {
+		val vaccinationEntry: VaccinationEntry? = dccHolder?.euDGC?.vaccinations?.firstOrNull()
+		binding.certificatePageTitle.setText(R.string.covid_certificate_title)
```

Comment:
Might this cause a flickering if we're setting a default title and then change it immediately?